src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HearingsHelper.java

```diff
@@ -187,15 +174,16 @@ private static void checkStartFinishTimes(List<String> errors, HearingDetailType
     /**
      * This finds the hearing by using the listed date and comparing it to the date provided. It streams through each
      * hearing and then checks whether the date listed is equal to the date provided.
-     * @param caseData used to query the hearing collection
+     *
+     * @param caseData    used to query the hearing collection
```

Comment:
double space here?

Reply:
Its a tab to get all the parameter descriptions inline. Its the way intellij likes it.